app/views/wikis/edit.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Edit Wiki</h1>
+
+<div class="row">
+	<div class="eight columns">
+		<%= form_for @wiki do |f| %>
+			<div class="form-group">
+				<%= f.label :title %>
+				<%= f.text_field :title, class: 'form-control', placeholder: "Enter wiki title" %>
+			</div>
+			<div class="form-group">
+				<%= f.label :body %>
+				<%= f.text_area :body, rows: 8, class: 'form-control', placeholder: "Enter wiki body" %>
+			</div>
+			<%= f.submit "Save", class: 'btn btn-success' %>
+		<% end %>
+	</div>
```

Comment:
Got some funky tabs to spaces going on here. :)

Reply:
I need to look into why it does that. I noticed that when I edit a file using a text editor it always ends up looking like this when I upload it. However, if I use vim to edit a file the spaces come out just right when uploaded.

Reply:
That's cause vim is the best :D